assume.go

```diff
@@ -48,24 +48,24 @@ func assumeRole(account, role string) {
 	role = adjustRoleName(role)
 
 	err := tryToAssumeRole(account, role)
-	if err != nil {
-		needToBeRotatedChan := make(chan bool)
-		go isNeedRotateKey(needToBeRotatedChan)
```

Comment:
Yeah, I realize that, however, as I described, that's not possible if you aren't able to gain access to the relevant IAM funtionality without a MFA session ;)